changelog.json

```
@@ -1,5 +1,14 @@
 {
   "releases": {
+    "1.6.3-beta3": [
+      "[New] Display \"pull with rebase\" if a user has set this option in their git config - #6553 #3422",
+      "[Added] Provide option to discard when files would be overwritten by a checkout - #6755. Thanks @mathieudutour!",
```

Tightening up this entry:

```suggestion
      "[Added] Option to discard when files would be overwritten on checkout - #6755. Thanks @mathieudutour!",
```